static/css/page-form.less

```
@@ -144,3 +144,76 @@ div#revertLink,
     float: left;
   }
 }
+
+// Styling for the new cover form accepting Copy/Pasted images
+
+.ol-cover-form {
+  margin-bottom: 20px;
+
+  .label {
+    margin-bottom: 10px;
+  }
+
+  .input {
+    display: flex;
+    gap: 10px;
+    align-items: center;
+  }
+
+  &--ia {
+    &_image {
+      height: 200px;
+    }
+  }
+
+  &--clipboard {
+    #image {
+      max-width: 100%;
+      max-height: 400px;
+      display: none;
+
+      &.visible {
+        display: block;
+      }
+    }
+
+    #hiddenFileInput {
+      display: none;
+    }
+
+    .paste-button {
+      width: auto;
+      &::after {
+        content: "No Image Pasted";
+      }
+    }
+
+    .upload-button {
+      width: auto;
+      display: none !important;
+      &.visible {
+        display: block !important;
+        &::after {
+          display: block;
+          content: "Use this image";
+        }
+        & + .paste-button::after {
+          content: "Change Image";
+        }
+      }
```

```suggestion
    .paste-button,
    .upload-button {
      width: auto;
    }
```
Now that we're setting the button text in the template (and updating client-side), there's no need for these `content` styling rules.  Styling for both buttons can also be consolidated.